lib/zendesk_apps_tools/command.rb

```diff
@@ -158,15 +148,20 @@ def server
     desc 'create', 'Create app on your account'
     shared_options
     method_option :zipfile, default: nil, required: false, type: :string
+    method_option :config, default: DEFAULT_CONFIG_PATH, required: false, aliases: '-c'
     def create
       cache.clear
       @command = 'Create'
 
       unless options[:zipfile]
-        app_name = JSON.parse(File.read(File.join options[:path], 'manifest.json'))['name']
+        app_name = manifest.name
       end
       app_name ||= get_value_from_stdin('Enter app name:')
       deploy_app(:post, '/api/v2/apps.json', name: app_name)
+      has_requirements = File.exist?(File.join(options[:path], 'requirements.json'))
+      say_status 'Install', 'installing'
+      install_app(has_requirements, app_id: cache.fetch('app_id'), settings: settings.merge(name: app_name))
```

Comment:
Shouldn't this be optional, especially considering it's new functionality being added to an existing command? Perhaps `zat create [-i | --install]`?